src/types.ts

```diff
@@ -25,3 +27,40 @@ export interface ImageRenderOptions extends RenderOptions {
   fullPageImage?: boolean;
   scaleImage?: number;
 }
+
+export type SanitizeRequest = {
+  filename: string;
+  content: string;
+  contentType?: string;
+  domPurifyConfig?: DOMPurify.Config;
+  allowAllLinksInSvgUseTags?: boolean;
+};
```

Comment:
Seems like this is never used, is there a reason to not remove it and remove the `V2` of `SanitizeRequestV2`?